heron/spi/src/java/com/twitter/heron/spi/utils/PackingTestUtils.java

```diff
@@ -74,13 +77,23 @@ public static PackingPlan testPackingPlan(String topologyName, IPacking packing)
 
   public static PackingPlan.ContainerPlan testContainerPlan(int containerId,
                                                             Integer... instanceIndices) {
+    List<Pair<String, Integer>> instanceInfo = new ArrayList<>();
+    for (int instanceIndex : instanceIndices) {
+      String componentName = "componentName-" + instanceIndex;
+      instanceInfo.add(new Pair<>(componentName, instanceIndex));
+    }
+    return testContainerPlan(containerId, instanceInfo);
+  }
+
+  public static PackingPlan.ContainerPlan testContainerPlan(int containerId,
+                                                            List<Pair<String, Integer>>
```

Comment:
If you change this signature to use `Pair<String, Integer>...` instead, you can pass an array or an unbounded number of Pairs, which is more flexible to the caller, since you don't need to populate a List or array first.